docs/generated/parameters/main.template.adoc

```diff
@@ -62,7 +62,7 @@
 [width="100%",cols="16%,11%,73%",options="header",]
```

Comment:
You can remove the contents of the generated/ folder as its generated during build phase by QS CI.

Reply:
Unfortunately while the PR was in review,  @tadrian88 is on vacation and he has the fork...